SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -1414,6 +1419,13 @@ class CVariable {
    */
   inline virtual bool SetPrimVar(unsigned long iPoint, CFluidModel *FluidModel) { return true; }
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] iPoint - Point index.
+   * \param[in] fluidmodel - fluid model.
+   */
+  inline virtual bool SetPrimVar(unsigned long iPoint, CNEMOGas *fluidmodel) {return false;}
```

Comment:
This is not a very good idea, NEMOGas is "convertible" to FluidModel, so the first overload can be called accidentally if you have a FluidModel pointer to a NEMOGas instance (which would be legal).
It would be safer to use the `SetPrimVar(unsigned long iPoint, CFluidModel *FluidModel)` signature also for the NEMO variables and then `static_cast` (or `dynamic_cast` if you want to error check) to NEMOGas